roles/cloud_connector/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+foreman_cloud_connector_url: https://foreman.example.com
+foreman_cloud_connector_username: admin
+foreman_cloud_connector_password: changeme
```

Comment:
if they are set in `vars`, they can't be overriden.

Reply:
Oops, I got this mixed up, again.